pkgs/by-name/vm/vmlinux-to-elf/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+python3.pkgs.buildPythonPackage {
+  pname = "vmlinux-to-elf";
+  version = "unstable-2024-07-20";
+
+  doCheck = false;
+
+  dependencies = builtins.attrValues {
+    inherit (python3.pkgs) zstandard lz4;
+  };
+
+  src = fetchFromGitHub {
```

Comment:
Per convention src should be placed earlier in the file, right after pname, version and pyproject (oh yeah, you should also add that)